Data-Structures/Queue/CircularQueue.js

```diff
@@ -77,7 +77,11 @@ class CircularQueue {
 
   // Displays the length of queue
   length() {
-    return this.queue.length - 1
+    if (this.queue.length > 0) {
```

Comment:
1. Why not use `this.checkEmpty()`?
2. Use a ternary: `this.checkEmpty() ? 0 : this.queue.length-1`.